lib/test/t_NegativeBinomialFactory_std.cxx

```diff
@@ -32,7 +32,7 @@ int main(int, char *[])
 
   try
   {
-    NegativeBinomial distribution(15, 0.7);
+    Polya distribution(15, 0.7);
     UnsignedInteger size = 10000;
     Sample sample(distribution.getSample(size));
     NegativeBinomialFactory factory;
```

Comment:
NegativeBinomialFactory must be renamed too

Reply:
Good point!